src/containers/my-resources/questions-container/QuestionsContainer.constants.tsx

```diff
@@ -9,23 +11,42 @@ import {
   Question,
   RemoveColumnRules,
   SortEnum,
+  QuestionTypesEnum,
   TableColumn,
   AdditionalPropsInterface
 } from '~/types'
 import { getFormattedDate } from '~/utils/helper-functions'
 import { styles } from '~/containers/my-resources/questions-container/QuestionsContainer.styles'
 
+export const questionsIcons = [
+  {
+    icon: <CheckCircleOutlineIcon />,
+    value: QuestionTypesEnum.MultipleChoice
+  },
+  {
+    icon: <NotesIcon />,
+    value: QuestionTypesEnum.OpenAnswer
+  },
+  {
+    icon: <RuleIcon />,
+    value: QuestionTypesEnum.OneAnswer
+  }
+]
+
 export const columns: TableColumn<Question>[] = [
   {
     label: 'myResourcesPage.questions.title',
     field: 'title',
     calculatedCellValue: (item: Question) => {
+      const questionIcon = questionsIcons.find(
+        (icon) => icon.value === item.type
+      )
       return (
         <IconTitleDescription
-          description={'Which word is the antonym of "benevolent"?'}
+          description={item.text}
           icon={
             <Box sx={styles.iconWrapper}>
-              <CheckCircleOutlineIcon />
+              {questionIcon ? questionIcon.icon : <CheckCircleOutlineIcon />}
```

Comment:
try to: questionIcon ?? <CheckCircleOutlineIcon />

Reply:
I think i should write it like this: questionIcon?.icon ?? <CheckCircleOutlineIcon />

Reply:
yeah, and maybe you should use just questionIcon?.icon without ??  ??<CheckCircleOutlineIcon />

Reply:
i think i should leave `??`. Because there should be some default value for icon and it is <CheckCircleOutlineIcon />.

Reply:
question type is required field, so it will always return an icon

Reply:
yeah, you are right